src/toil/leader.py

```diff
@@ -34,6 +34,15 @@
 
 logger = logging.getLogger( __name__ )
 
+# create new logger for worker output logs so we can format them appropriately
+outputLogger = logging.getLogger('workerOutput')
```

Comment:
I think this should be `__name__ + ".workerOutput" such that it is a child of the "toil" logger.